src/it/scala/com/wavesplatform/it/BlacklistTestSuite.scala

```diff
@@ -26,7 +26,7 @@ class BlacklistTestSuite extends FreeSpec with Matchers with BeforeAndAfterAll w
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
-    Await.result(Future.traverse(nodes)(_.waitForPeers(nodes.size - 1)), 2.minute)
+    nodes.foreach(_.status) // Initialize
```

Comment:
Why no `Await` here?

Reply:
I've replace it by the logging a number of nodes. This line is required to initialize the lazy variable only. So, we haven't to do it